### ISSUE TITLE
Datenpunkt state.lockState geht nicht auf unlatched oder unlatching.

### ISSUE THREAD

**Issue by MarkusKress** [iobroker-community-adapters/ioBroker.nuki-extended#227]
Ich nutze Nuki-Extended 2.4 (install from git) mit einem Nuki 3.0, ext. Bridge, Door-Sensor und Keypad.
Der Datenpunkt "nuki-extended.0.smartlocks.<türname>.state.lockState" nimmt zwar z.B. die Werte "1" (locked) und "3" (unlocked) ein, nicht jedoch "5" (unlatched) oder z.B. "7" (unlatching). D.h. ich kann das Öffnen der Tür mit der Falle nicht ablesen. Ist dies normal (gibt die API das nicht her) oder ist dies ein Bug?

**Comment by Grothesk242** [comment]
Ich frade das anders ab: 

(getState("nuki-extended.0.smartlocks.wohnungstür.state.lastAction").val == 3

Hier ist 3 Unlatching.

**Comment by MarkusKress** [comment]
Ja, diesen Datenpunkt habe ich auch gesehen und hätte diesen dann auch
verwendet

Grothesk242 ***@***.***> schrieb am Fr., 22. Apr. 2022, 19:20:

**Comment by Grothesk242** [comment]
:-( Leider gibt es den Datenpunkt 
state.lastAction
wohl nicht mehr. Ist das in der API rausgefallen?
